### DISCUSSION TITLE
Laravel Web development

### DISCUSSION THREAD

**Discussion by fmahadyBD** [community/community#101554] [Programming Help] [answered]
### Select Topic Area

Product Feedback

### Body

I want to find someone for learning Laravel web development. I am new in Laravel. I am starting to create the Laravel ecommerce website . Is this are good idea to learn?

**Comment by ettaboyle** [comment, marked as answer]
Welcome to the GitHub Community, @fmahadyBD, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it from `General` to `Programming Help` for you. Good luck!

**Comment by Dakotas66s** [comment]
I've been there, too, starting from scratch with Laravel. It's beginner-friendly and offers tons of learning resources. I suggest diving into tutorials, practicing coding, and joining forums like Laracasts for support. 
By the way, I need help developing a website for my work. My friend advised me to check out these [Custom software development](https://sysint.net/services/custom-software-development-for-startups.html) services. Any additional advice from you, guys? Thanks!

**Reply by fmahadyBD** [reply]
Thank you for sharing you experience with me
